app/views/content/funding-and-support/salaried-teacher-training.md

```diff
@@ -41,19 +41,27 @@ quote:
 
 There are some teacher training courses that come with a salary. 
 
-This means you will not have to pay tuition fees and will receive a salary while you train to get [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts). This is what you need to teach in many primary, secondary and special schools in England. 
+This means you'll be paid a salary while you train to get [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts). You'll need QTS to teach in many primary, secondary and special schools in England. 
 
-The main salaried courses include School Direct salaried, postgraduate teaching apprenticeships (PGTA) and teacher training delivered by Teach First.
+Some salaried teacher training courses may also come with fees. Check with individual providers for more information. 
+
+The salaried courses are:
+
+* School Direct salaried
+* postgraduate teaching apprenticeships (PGTA)
+* Teach First teacher training
+
+If you do not already have a degree, you may be eligible for a [teacher degree apprenticeship (TDA)](/train-to-be-a-teacher/teacher-degree-apprenticeships). A TDA is an undergraduate teacher training course where you’ll work in a school and earn a salary while getting a degree and QTS.
```

Comment:
Is it worth moving this to the `Am I eligible section`, perhaps under an h3 of `If you don't have a degree` or similar?